src/components/NumberInput/NumberInput.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { classNames } from "../../utils";
+import styles from "./styles.css";
+import Input from '../Input/Input.jsx';
+
+const NumberInput = ({ type, label, min, mode, id, error, ...props }) => {
+  return (
+    <Input 
+    	type={type}
+    	id={id}
+    	error={error}
+    	min={min}
+    	mode={mode}
```

Comment:
Shouldn't it be "numeric" by the default and not changeable since it will always be a Number input?